mono/mini/aot-runtime.c

```diff
@@ -1951,10 +1960,26 @@ load_aot_module (MonoAssembly *assembly, gpointer user_data)
 		return;
 
 	mono_aot_lock ();
+
+	if (container_assm_name && !container_amodule) {
+		char *local_ref = container_assm_name;
+		container_assm_name = NULL;
+
+		MonoImageOpenStatus status = MONO_IMAGE_OK;
+		gchar *dll = g_strdup_printf ("%s.dll", local_ref);
+		MonoAssembly *assm = mono_assembly_open_predicate (dll, FALSE, FALSE, NULL, NULL, &status);
```

Comment:
Oh right. `mono_assembly_open_a_lot`.  (mainly I'd prefer to minimize the number of callers of mono_assembly_open_predicate - it's the most general form and hardest to audit)